server-common/src/main/java/org/apache/kafka/server/common/serialization/NodeToControllerChannelManager.java

```diff
@@ -15,10 +15,11 @@
  * limitations under the License.
  */
 
-package org.apache.kafka.server;
+package org.apache.kafka.server.common.serialization;
```

Comment:
not sure the connection between `NodeToControllerChannelManager` and serialization. Maybe `org.apache.kafka.server.common` is good enough?